doc/src/manual/arrays.md

```diff
@@ -801,6 +844,11 @@ julia> sparse(eye(5))
   [3, 3]  =  1.0
   [4, 4]  =  1.0
   [5, 5]  =  1.0
+
+julia> sparse([1.0, 0.0, 1.0])
+3-element SparseVector{Float64,Int64} with 2 stored entries:
+  [1]  =  1.0
+  [3]  =  1.0
 ```
 
 You can go in the other direction using the [`full()`](@ref) function. The [`issparse()`](@ref)
```

Comment:
(We should probably remove this mention of `full`, and instead suggest `Array`.)